flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/runtime/functions/SqlFunctionUtils.java

```diff
@@ -1063,8 +1063,8 @@ public static String toBase64(byte[] bytes){
 		return Base64.getEncoder().encodeToString(bytes);
 	}
 
-	public static byte[] fromBase64(BinaryString bs){
-		return Base64.getDecoder().decode(bs.getBytes());
+	public static String fromBase64(BinaryString bs){
+		return new String(Base64.getDecoder().decode(bs.getBytes()));
```

Comment:
We can not use `new String` directly, because we don't know the default coder must be UTF-8.
Maybe we can use `BinaryString.fromBytes`?

Reply:
Do you mean we return `BinaryString` for `fromBase64`, or just like `BinaryString.fromBytes(...).toString()` ?

Or we can make it explicit UTF8 like `CAST` in ScalarOperatorGens:L942 ?

Reply:
Yes, Using `BinaryString` is better.

Reply:
Good idea. Done.